tests/tools/conftest.py

```diff
@@ -0,0 +1,34 @@
+from typing import TYPE_CHECKING
```

Comment:
I am not smart enough to see what actually are testing here. Maybe you can add a comment

Reply:
This is for type annotations only. It has nothing to do with testing, but rather to support IDEs as well as set the package up for future testing with MyPy (see #223).

Reply:
Thanks for the explanation. I think it would be good to have as a module docstring.

Reply:
This will be littered all over the code those in the future - Maybe I'll add it to the developer's documentation instead? Or do you want a comment in all module doc-strings where it'll be?

Reply:
Added a new documentation page about testing and tools for developers :)